### ISSUE TITLE
[Heartbeat] Service activity logging mode

### ISSUE THREAD

**Issue by andrewvc** [elastic/beats#29901]
For some uses, e.g. the synthetics service, it is necessary to log, in real time, basic information about the work heartbeat is doing. This data should all be:

1. Free of sensitive data
2. Include enough information to provide a basic picture of usage
3. Provide a basic picture of performance

We can accomplish this by adding an event after each monitor run:

```js
{
  "event": {"action": "monitor.ran"}, 
  "monitor": {
    "type": "http", 
    "duration": {"ms": "713897312"}, // duration in millis
  }
}
```

**Comment by andrewvc** [comment]
Related https://github.com/elastic/beats/issues/30433 . 

@paulb-elastic TBH, I'd prefer we name our units where possible and switch to millis. Micros are very inconvenient to deal with

**Comment by vigneshshanmugam** [comment]
Closed by #31405

**Comment by emilioalvap** [comment]
PostFF LGTM.
